django_slack/logs.py

```diff
@@ -0,0 +1,84 @@
+import copy
+import logging
+from . import slack_message
+from django.conf import settings
+from django.views.debug import ExceptionReporter
+
+
+class SlackExceptionHandler(logging.Handler):
+    """
+    An exception log handler that sends log entries to a Slack channel.
+    """
+
+    def __init__(self, **kwargs):
+        # pop any kwargs that shouldn't be passed into the slack message attachment here.
+        self.template = kwargs.pop('template', 'django_slack/exception')
+        self.kwargs = kwargs
+        logging.Handler.__init__(self)
+
+    def emit(self, record):
+        try:
+            request = record.request
+            subject = '%s (%s IP): %s' % (
+                record.levelname,
+                ('internal' if request.META.get('REMOTE_ADDR') in settings.INTERNAL_IPS
+                 else 'EXTERNAL'),
+                record.getMessage()
+            )
+        except Exception:
+            subject = '%s: %s' % (
+                record.levelname,
+                record.getMessage()
```

Comment:
I kept this as it was implemented in the Django AdminEmailHandler, but can add a trailing comma if preferred.